appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/java/org/kie/appformer/formmodeler/codegen/view/impl/java/RoasterViewSourceGenerator.java

```diff
@@ -37,137 +32,182 @@
+    public RoasterViewSourceGenerator(final Instance<InputCreatorHelper<? extends FieldDefinition>> creatorInstances) {
         this.creatorInstances = creatorInstances;
     }
 
     @PostConstruct
     protected void init() {
-        for ( final InputCreatorHelper helper : creatorInstances ) {
-            creatorHelpers.put( helper.getSupportedFieldTypeCode(),
-                    helper );
+        for (final InputCreatorHelper helper : creatorInstances) {
+            creatorHelpers.put(helper.getSupportedFieldTypeCode(),
+                               helper);
         }
     }
 
     @Override
-    public String generateJavaSource( final SourceGenerationContext context ) {
+    public String generateJavaSource(final SourceGenerationContext context) {
         final JavaClassSource viewClass = createClassSource();
-        final String packageName = getPackageName( context );
-
-        addTypeSignature( context, viewClass, packageName );
-        addImports( context, viewClass );
-        addAnnotations( context, viewClass );
-        addAdditional( context, viewClass );
-
-        addBaseViewFieldsAndMethodImpls( context, viewClass );
+        final String packageName = getPackageName(context);
+
+        addTypeSignature(context,
+                         viewClass,
+                         packageName);
+        addImports(context,
+                   viewClass);
+        addAnnotations(context,
+                       viewClass);
+        addAdditional(context,
+                      viewClass);
+
+        addBaseViewFieldsAndMethodImpls(context,
+                                        viewClass);
         return viewClass.toString();
     }
 
     protected JavaClassSource createClassSource() {
-        return Roaster.create( JavaClassSource.class );
+        return Roaster.create(JavaClassSource.class);
     }
 
-    protected abstract void addAdditional( SourceGenerationContext context,
-            JavaClassSource viewClass );
+    protected abstract void addAdditional(SourceGenerationContext context,
+                                          JavaClassSource viewClass);
 
-    protected void addBaseViewFieldsAndMethodImpls( final SourceGenerationContext context,
-            final JavaClassSource viewClass ) {
-        final StringBuffer readOnlyMethodSrc = new StringBuffer(  );
+    protected void addBaseViewFieldsAndMethodImpls(final SourceGenerationContext context,
+                                                   final JavaClassSource viewClass) {
+        final StringBuffer readOnlyMethodSrc = new StringBuffer();
 
-        for ( final FieldDefinition fieldDefinition : context.getFormDefinition().getFields() ) {
+        for (final FieldDefinition fieldDefinition : context.getFormDefinition().getFields()) {
 
-            if ((fieldDefinition.isAnnotatedId() && !displaysId()) || isBanned( fieldDefinition ))
+            if ((fieldDefinition.isAnnotatedId() && !displaysId()) || isBanned(fieldDefinition)) {
```

Comment:
@alessioscalici isAnnotatedI() method has been removed from FieldDefinition in recent pull-request of @pefernan (field definitions are not created for annotated ids anymore), so that's probably why the jenkins builds are failing.